src/System.Private.CoreLib/src/System/Text/DecoderNLS.cs

```diff
@@ -137,10 +137,10 @@ internal unsafe override int GetCharCount(byte* bytes, int count, bool flush)
 
             // Just call pointer version
             fixed (byte* pBytes = &bytes[0])
-                fixed (char* pChars = &chars[0])
-                    // Remember that charCount is # to decode, not size of array
-                    return GetChars(pBytes + byteIndex, byteCount,
-                                    pChars + charIndex, charCount, flush);
+            fixed (char* pChars = &chars[0])
```

Comment:
I believe that the corefx coding style allows both more compact:

```
fixed (byte * p1 = ...)
fixed (byte * p2 = ...)
fixed (byte * p3 = ...)
{
    ... do something with p1, p2, p3 ...
}
```

and more verbose style like:

```
fixed (byte * p1 = ...)
{
    fixed (byte * p2 = ...)
    {
        fixed (byte * p3 = ...)
        {
            ... do something with p1, p2, p3 ...
        }
    }
}
```

I remember it being discussed during codereviews. For example, you can see the more compact style here: https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Logging/NetEventSource.Common.cs#L530